packages/studio/frontend/src/views/FlowEditor/common/EditableInput.jsx

```diff
@@ -0,0 +1,80 @@
+import classnames from 'classnames'
```

Comment:
If possible, can you convert this `.jsx` file to a `.tsx` one as we want to get rid of the former?